### PR TITLE
[runtime-security] add flare and status

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR adds flare and status for the runtime component of the security-agent.
It also move the cmd_port in the config from the compliance section to a common place

### Motivation

What inspired you to submit this pull request?

### Additional Notes

It currently report the status of the connection between the agent and system-probe and the total of event received.

### Describe your test plan

Just run the `security-agent status` a `Runtime Security` is now present

### CODE REVIEW

File: pkg/status/templates/runtimesecurity.tmpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Runtime Security
+================
+
+{{- if not .RuntimeSecurityStatus}}
+  Not enabled
+{{- else}}
+  {{- with .RuntimeSecurityStatus}}
+  Connected: {{.connected}}
+  Total Event received: {{.eventReceived}}
```
Comment (xornivore) [inline pkg/status/templates/runtimesecurity.tmpl:9]:
```suggestion
  Events received: {{.eventReceived}}
```

File: cmd/security-agent/app/status.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ func requestStatus() error {
 	var e error
 	var s string
 	c := util.GetClient(false) // FIX: get certificates right then make this true
-	urlstr := fmt.Sprintf("https://localhost:%v/agent/status", config.Datadog.GetInt("compliance_config.cmd_port"))
+	urlstr := fmt.Sprintf("https://localhost:%v/agent/status", config.Datadog.GetInt("security_agent_config.cmd_port"))
```
Comment (xornivore) [inline cmd/security-agent/app/status.go:71]: We may have a conflicting change, let's use `security_agent.cmd_port` for this.

I'll rebase https://github.com/DataDog/datadog-agent/pull/6112 if your PR goes in first.

File: pkg/status/templates/runtimesecurity.tmpl [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+================
+
+{{- if not .RuntimeSecurityStatus}}
+  Not enabled
+{{- else}}
+  {{- with .RuntimeSecurityStatus}}
+  Connected: {{.connected}}
+  Events received: {{.eventReceived}}
+  {{- end }}
+{{- end }}
```
Comment (xornivore) [inline pkg/status/templates/runtimesecurity.tmpl:11]: Nit, missing eol.

**Review by xornivore — APPROVED** [review]

**Review by lebauce — APPROVED** [review]

**Review by olivielpeau — APPROVED** [review]
only reviewed agent-core-owned files, LGTM

**Pull request merged**